modules/yahooAdvertisingBidAdapter.js

```diff
@@ -6,9 +6,9 @@ import { Renderer } from '../src/Renderer.js';
 import {hasPurpose1Consent} from '../src/utils/gpdr.js';
 
 const INTEGRATION_METHOD = 'prebid.js';
-const BIDDER_CODE = 'yahoossp';
+const BIDDER_CODE = 'yahooAdvertising';
```

Comment:
this will be quite annoying for publishers, as the key names in gam are restricted to 20 characters

![image](https://github.com/prebid/Prebid.js/assets/1683175/5258e33c-45ed-4a17-99d5-fc48ac0d320b)

Reply:
Yes, I thought this too. We will go back to product with this issue to get their thoughts on it